From 92f20fec63bdbb7341135875f6579b148ac07574 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Tue, 11 Apr 2017 18:45:01 +0100
Subject: [PATCH 2/4] gpu: midgard: drop "clk_mali" name

As the "clock-names" entry has been removed from the device tree, also
remove the clock name "clk_mali" in the driver to get the only clock
without any name.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>
---
 drivers/gpu/arm/midgard/mali_kbase_core_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mali_kbase_core_linux.c b/mali_kbase_core_linux.c
index 6cba3e2318ed..d35ca2bb446c 100644
--- a/mali_kbase_core_linux.c
+++ b/mali_kbase_core_linux.c
@@ -3269,7 +3269,7 @@ static int power_control_init(struct platform_device *pdev)
 	}
 #endif /* LINUX_VERSION_CODE >= 3, 12, 0 */

-	kbdev->clock = clk_get(kbdev->dev, "clk_mali");
+	kbdev->clock = clk_get(kbdev->dev, NULL);
 	if (IS_ERR_OR_NULL(kbdev->clock)) {
 		err = PTR_ERR(kbdev->clock);
 		kbdev->clock = NULL;
--
2.11.0
